packages/lodestar/src/validator/validator.ts

```diff
@@ -60,6 +61,8 @@ class Validator {
       this.rpcClient = this.opts.rpcInstance;
     } else if(this.opts.rpc) {
       this.rpcClient = new RpcClientOverWs({rpcUrl: this.opts.rpc}, {config: this.config});
+    } else if(this.opts.restUrl) {
```

Comment:
I would move api client initialization to separate method and probably rename it to api or apiClient or something more generic than rpcClient